deployment/docker-compose.prod.yaml

```diff
@@ -0,0 +1,54 @@
+version: "3"
+services:
+
+  nestjs:
+    container_name: nestjs
+    build:
+      context: ../
+      dockerfile: ./deployment/Dockerfile.prod
+    depends_on:
+      - postgres
+    env_file:
+      - .env.prod
+    expose:
+      - 3000
+
+  postgres:
+    container_name: postgres
+    image: postgres:10.4
+    ports:
+      - "5432:5432"
```

Comment:
Should map to TYPEORM_PORT, or TYPEORM_PORT should not be part of .env file

Reply:
I don't know if we can use an env value in the port definition, so we could remove these var and let the port definition in the docker compose file.

Reply:
I believe you can. But OK for now, hardcoding the port number in the docker-compose.yaml file seems fine to me.